### ISSUE TITLE
Custom command adding new lines before each argument

### ISSUE THREAD

**Issue by michaelyatrakos** [gocd/gocd#7662]
##### Issue Type

- Bug Report

##### Environment

```plain
"Go Server Information": {
    "Version": "19.12.0 (10888-29b0f854605987c8edab9dced4814b62dc751a11)"
}
"OS Information": {
    "OS Name": "Linux",
    "OS Version": "3.10.0-1062.9.1.el7.x86_64",
    "System Architecture": "amd64",
    "Available Processors": 4,
    "Average System Load": 0.63
},
```
###### Basic environment details

* Go Version: `19.12.0`
* JAVA Version: `13.0.1`
* OS: `Linux 3.10.0-1062.9.1.el7.x86_64`

##### Steps to Reproduce

1. Add a task using a custom command
2. Add at least two arguments

##### Expected Results
The custom command should run with both arguments provided, similar to:
`command arg1 arg2 arg3`

##### Actual Results
The command is being run with all args following the first appearing on new lines, similar to:
```
command arg1
arg2
arg3
```
##### Log snippets
The custom command I'm using in this instance is maven `mvn`. This is the logs from the agent run:

```
[go] Job Started: 2020-01-27 13:54:49 UTC
[go] Start to prepare Elastic-APM-Java-Agent/1/Continuous-Integration/17/Compile on build.servers.iminsys.com [/var/lib/go-agent]
[go] Start to build Elastic-APM-Java-Agent/1/Continuous-Integration/17/Compile on build.servers.iminsys.com [/var/lib/go-agent]
[go] Task: mvn clean took: 3.681s exited: 1
 compile
 test
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
. . .
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.266s
[INFO] Finished at: Mon Jan 27 13:54:58 UTC 2020
[INFO] Final Memory: 10M/40M
[INFO] ------------------------------------------------------------------------
[ERROR] Unknown lifecycle phase "clean
[ERROR] ". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-site, site, post-site, site-deploy, pre-clean, clean, post-clean. -> [Help 1]
[ERROR] 
```

##### Code snippets/Screenshots
This is the custom command in the UI:

![image](https://user-images.githubusercontent.com/49722201/73180593-458e4f00-411e-11ea-8ecc-d072c8ff0b8c.png)

And this is the corresponding command in the config XML:
```
<job name="Compile">
	<tasks>
		<exec command="mvn">
			<arg>clean</arg>
			<arg>compile</arg>
			<runif status="passed" />
		</exec>
	</tasks>
</job>
```

##### Any other info

The go-agent does has access to the mvn command and this is not unique to using the mvn command, I have reproduced the issue with several other commands.

**Comment by arvindsv** [comment]
@michaelyatrakos This seems to be a duplicate of https://github.com/gocd/gocd/issues/7557. Can you try what is mentioned there and see if it helps?

A workaround is mentioned here: https://github.com/gocd/gocd/issues/7557#issuecomment-574776093

This is fixed in the upcoming release of GoCD (which should be soon).

**Comment by kritika-singh3** [comment]
There is a fix pushed in the GoCD release 20.1.0 which was released recently. Please upgrade and verify that the issue has been fixed.

I'll be closing this issue for now. Please reopen/comment if that is not the case.
